packages/minifier-js/minifier.js

```diff
@@ -29,14 +30,11 @@ meteorJsMinify = function (source) {
     // Although Babel.minify can handle a wider variety of ECMAScript
     // 2015+ syntax, it is substantially slower than UglifyJS, so we use
     // it only as a fallback.
-    if (Babel.getMinifierOptions) {
-      var options = Babel.getMinifierOptions({
-        inlineNodeEnv: NODE_ENV
-      });
-      result.code = Babel.minify(source, options).code;
-    } else {
-      result.code = Babel.minify(source).code;
-    }
+    var options = Babel.getMinifierOptions({
+      inlineNodeEnv: NODE_ENV
+    });
+    result.code = Babel.minify(source, options).code;
+    result.minifier = 'babeli';
```

Comment:
[The official name of the minifier is `babel-minify`.](https://github.com/babel/minify#note) :slightly_smiling_face:

Reply:
Ah, it is very often called babeli, but sticking to the official name would be better indeed.